imagepipeline/src/main/java/com/facebook/imagepipeline/producers/HttpUrlConnectionNetworkFetcher.java

```diff
@@ -75,16 +76,23 @@ public void onCancellationRequested() {
   @VisibleForTesting
   void fetchSync(FetchState fetchState, Callback callback) {
     HttpURLConnection connection = null;
-
+    InputStream is = null;
     try {
       connection = downloadFrom(fetchState.getUri(), MAX_REDIRECTS);
 
       if (connection != null) {
-        callback.onResponse(connection.getInputStream(), -1);
+        is = connection.getInputStream();
+        callback.onResponse(is, -1);
       }
     } catch (IOException e) {
       callback.onFailure(e);
     } finally {
+      if (is != null) {
+        try {
+          is.close();
+        } catch (IOException e) {
```

Comment:
would be nice to have a short comment stating that we knowingly ignore the `IOException`